### PR TITLE
"has handshake accepted" indicator

### PR DESCRIPTION
**Pull request by mjoyce91**
Use with https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/585

Displays a red indicator on the candidates list if the bidder has accepted a handshake for a cycle position in an active bid cycle.

### CODE REVIEW

File: src/Constants/Tooltips.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
-// eslint-disable-next-line import/prefer-default-export
 export const DECONFLICT_TOOLTIP_TEXT = {
   title: 'Deconflict Alert',
-  text: 'Candidate Ranked #1 on, at least, one other Short List',
+  text: 'Candidate Ranked #1 on at least one other Short List',
```
Comment (elizabeth-jimenez) [inline src/Constants/Tooltips.js:3]: 😭

File: src/Components/BureauPage/PositionManagerBidders/PositionManagerBidders.jsx [hunk 2]
```diff
@@ -274,19 +275,35 @@ class PositionManagerBidders extends Component {
...
-          tabIndex="0"
-          interactive
-          style={{ height: 'fit-content' }}
-          useContext
-        >
-          <FA name={'exclamation-triangle'} className={'deconflict-indicator'} />
-        </Tooltip>
-        : '',
+      Deconflict: (
+        <div className="alert-indicators">
```
Comment (elizabeth-jimenez) [inline src/Components/BureauPage/PositionManagerBidders/PositionManagerBidders.jsx:279]: is this class name for usability?

**Review by elizabeth-jimenez — APPROVED** [review]
🍰 👍 Looks really good!

File: src/Components/BureauPage/PositionManagerBidders/PositionManagerBidders.jsx (hunk 2 above)
Comment (mjoyce91) [reply src/Components/BureauPage/PositionManagerBidders/PositionManagerBidders.jsx:279]: Yeah just in case we ever decide to style it

**Pull request merged**